Adyen/Model/BrowserInfo.swift

```diff
@@ -32,10 +35,21 @@ public struct BrowserInfo: Encodable {
             completion(BrowserInfo(userAgent: cachedUserAgent))
         }
     }
+
+    public func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encode(acceptHeader, forKey: .acceptHeader)
+        try container.encodeIfPresent(userAgent, forKey: .userAgent)
+    }
```

Comment:
why is this needed? the default implementation should work, right?

Reply:
Same as [previous](https://github.com/Adyen/adyen-ios/pull/713#discussion_r754027972) question:
For some reason it crashes 🤷‍♂️